jest-setup.ts

```diff
@@ -34,8 +34,4 @@ jest.mock( './src/hooks/use-offline', () => ( {
 	useOffline: jest.fn().mockReturnValue( false ),
 } ) );
 
-jest.mock( 'strip-ansi', () => ( {
-	default: jest.fn().mockImplementation( ( str: string ) => str ),
-} ) );
```

Comment:
Relocated to the `__mocks__` directory to co-locate third-party mocks and make the mock universal for all tests.